sktime/utils/validation/tests/test_write_dataframe_to_ts.py

```diff
@@ -0,0 +1,41 @@
+__author__ = ["Jason Pong"]
+__all__ = ["test_write_dataframe_to_ts_success", "test_write_dataframe_to_ts_fail"]
+
+import os
+
+import numpy as np
+import pandas as pd
+import pytest
+from _pytest.tmpdir import tmp_path
```

Comment:
I don't think we need this import, pytest automatically recognises `tmp_path` as a test fixture see https://docs.pytest.org/en/stable/tmpdir.html